pymc/logprob/transforms.py

```diff
@@ -672,7 +672,10 @@ def find_measurable_transforms(fgraph: FunctionGraph, node: Node) -> Optional[Li
 
     # Do not apply rewrite to discrete variables
     if measurable_input.type.dtype.startswith("int"):
-        return None
+        if check_negation(node.op.scalar_op, node.inputs[0]) is False and not isinstance(
+            node.op.scalar_op, Add
+        ):
+            return None
```

Comment:
Perhaps it's more readable like this?
```suggestion
        if not (
            check_negation(node.op.scalar_op, node.inputs[0]) 
            or isinstance(node.op.scalar_op, Add)
        ):
            return None
```